client/ayon_core/hosts/max/api/pipeline.py

```diff
@@ -242,3 +242,20 @@ def get_previous_loaded_object(container: str):
         if str(obj) in sel_list:
             node_list.append(obj)
     return node_list
+
+
+def remove_container_data(container: str):
```

Comment:
I think it is `node_name` instead of `container`, or?
```suggestion
def remove_container_data(node_name: str):
```

Reply:
maybe `container_node` is better as it is removing the container node which stores the container's data for upversioning and switching assets in scene inventory